sources/dylan/apple-dylan-test-suite/test-range.dylan

```diff
@@ -50,11 +50,11 @@ end test range-4;
 // not except through and up-to in a single expression
 //
 define test range-5 (description: "Through:, by:, and size:")
```

Comment:
That's what that description implies, but there's no such ``through:`` keyword anymore. I'll update the description and remove the comments above the test too, which don't apply anymore.

I'll add also correct cases too